Sources/JNI/SwiftJNI.swift

```diff
@@ -284,6 +308,17 @@ extension JNI {
 
         return strings
     }
+
+    public func GetObjectArrayElement(in array: JavaObjectArray, at index: Int) throws -> JavaObject {
```

Comment:
I wonder whether we can make swift wrapper types for typed arrays

struct JavaArray<T: JavaType> {
  subscript(index: Int) -> T
}

Would leave this for another PR though